manim/utils/cfgwriter.py

```diff
@@ -26,7 +27,9 @@ def check_valid_colour(color):
 def main():
     console = Console()
     config = configparser.ConfigParser()
-    config.read("manim.cfg")
+    config.read(
+        path.abspath(path.join(__file__,"../../default.cfg"))
```

Comment:
Ok now do you expect this change the `default.cfg` and I expect this should not be done?

Reply:
This... won't change `default.cfg` at all.

It will read from it, and write what modifications the users had made to it to `manim.cfg`

Reply:
I prefer using some other function which has been defined in `utils/cfgwriter.py`. Use it rather than manually adding.

Reply:
What?
Nothing is being manually added here though.

Reply:
Use `successfully_read_files ` This has the precedence. Import it from `utils/cfgwriter.py`.

Reply:
```suggestion
        successfully_read_files
```

Reply:
That doesn't apply here sorry.